tests/test_core.py

```diff
@@ -3900,7 +3899,7 @@ def test_dylink_i64(self):
       int64_t sidey() {
         return 42;
       }
-    ''', 'other says 42.', force_c=True, main_module=2)
+    ''', 'other says 42.', force_c=True, main_module=1)
```

Comment:
please add comments for these tests as to why they need mode 1 (the two at the end have comments, but not several of the others)

Reply:
Done